OpenRA.Game/Server/Connection.cs

```diff
@@ -26,6 +27,7 @@ public class Connection : IDisposable
 		public readonly int PlayerIndex;
 		public readonly string AuthToken;
 		public readonly EndPoint EndPoint;
+		public readonly Stopwatch ConnectionTimer = Stopwatch.StartNew();
```

Comment:
I'm not a big fan either on having this here too, it would be better to have something listening on ConnectionStateChanged or solve it in another way.

Reply:
This is the server's Connection, so we dont have `ConnectionStateChanged` etc!

Reply:
Doh, then this makes more sense!